ChangeHistory

```diff
@@ -6,6 +6,7 @@ Releases are tagged on the 'master' branch as "g4cmp-Vxx-yy-zz".
 Features and bug fixes are tagged on the 'develop' branch using the JIRA
 ticket identifier, "G4CMP-nnn", or using the GitHub pull request, "PR-nn".
 
+2024-10-11  G4CMP-407 : Calculate l0 from physical constants.
```

Comment:
When you remove the G4CMP-404 line (below), update this to today's date.

Reply:
Done!